src/utils/formatSelector.js

```diff
@@ -221,8 +221,10 @@ export const formatSelector = (
           size *= 4 // vsize
           // Varint witness items length.
           size += 1
-          // Calculate vsize
-          size = ((size + witScale - 1) / witScale) | 0
+          // Calculate vsize. Round up instead of down if it's a corner case 1in/1out tx.
+          specialCase
+            ? (size = Math.ceil((size + witScale) / witScale))
```

Comment:
Please remove the special case and just use this line.
Adding this 1 byte just in case is what we should have done anyway.